frontend/src/components/eMIB/EditTask.jsx

```diff
@@ -122,7 +138,29 @@ class EditTask extends Component {
               <label htmlFor="your-tasks-text-area" style={styles.tasks.title}>
                 {LOCALIZE.emibTest.inboxPage.addEmailTask.task}
               </label>
-              <i className="fas fa-question-circle" style={styles.tasks.icon} />
+              <OverlayTrigger
+                trigger="focus"
+                placement="right"
+                overlay={
+                  <Popover id="task-tooltip" style={styles.tasks.tooltipContainer}>
+                    <div>
+                      <p style={styles.tasks.tooltipContent}>
+                        {LOCALIZE.emibTest.inboxPage.taskContent.taskTooltipPart1}
+                      </p>
+                      <p style={styles.tasks.tooltipContent}>
+                        {LOCALIZE.emibTest.inboxPage.taskContent.taskTooltipPart2}
+                      </p>
+                    </div>
+                  </Popover>
+                }
+              >
+                <i
+                  id="task-tooltip"
+                  tabIndex="0"
```

Comment:
if this has a tab index, it probably needs an aria label or alt tag since it's clickable?

Reply:
Added aria labels in commit #https://github.com/code-for-canada/project-thundercat/pull/214/commits/ec70e900d19d1da50654be88cb31d87b7897b14b